tuned/verifylog.py

```diff
@@ -0,0 +1,108 @@
+
+import json
+
+class VerifyLog:
+
+	obj = None
+
+	def __init__(self, log={}):
+		self.vLog = log
+		return
+
+	@classmethod
+	def get_obj(self, log={}):
+		"""
+		Create and return a singleton object.
+		"""
+		if (not self.obj):
+			self.obj = VerifyLog(log)
+		return self.obj
+
+	def add_log(self, iname, log={}):
+		if (iname not in self.vLog.keys()):
+			self.vLog[iname] = []
+		if (log):
+			self.vLog[iname].append(log)
+
+		return
+
+	def get_log(self):
+		return self.vLog
+
+	def put_log(self):
+		vlog = self.vLog
+		self.vLog = {}
+		return vlog
+
+	def _display_device(self, log):
+		ret = True
+		for i in range(len(log)):
+			for cmd in list(log[i].keys()):
+				print(" %s=%s" % (cmd, log[i][cmd]["value"]))
+				for dev in log[i][cmd]["devices"]:
+					for k in dev.keys():
```

Comment:
Similar to above, why not use "for k, v in dev.items()"? Also similar in other functions down there.